dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/patch/operation/DSpaceObjectMetadataRemoveOperation.java

```diff
@@ -91,9 +99,12 @@ private void remove(Context context, DSpaceObject dso, DSpaceObjectService dsoSe
             throw new IllegalArgumentException("This index (" + index + ") is not valid number.", e);
         } catch (ArrayIndexOutOfBoundsException e) {
             throw new UnprocessableEntityException("There is no metadata of this type at that index");
-        } catch (SQLException e) {
-            throw new DSpaceBadRequestException("SQLException in DspaceObjectMetadataRemoveOperation.remove " +
-                    "trying to remove metadata from dso.", e);
+        } catch (SQLException ex) {
+            String msg;
```

Comment:
why not `String msg = `?

Reply:
The log removed because it is not requeired